#!/usr/bin/env python
PACKAGE = "aero_control"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("x_linear_P", double_t, 0, "X Linear P Term", 0, 0, 5)
gen.add("x_linear_I", double_t, 0, "X Linear I Term", 0, 0, 5)
gen.add("x_linear_D", double_t, 0, "X Linear D Term", 0, 0, 5)
gen.add("x_gain", double_t, 0, "X Gain", 1, 0, 5)
gen.add("y_linear_P", double_t, 0, "Y Linear P Term", 0, 0, 5)
gen.add("y_linear_I", double_t, 0, "Y Linear I Term", 0, 0, 5)
gen.add("y_linear_D", double_t, 0, "Y Linear D Term", 0, 0, 5)
gen.add("y_gain", double_t, 0, "Y Gain", 1, 0, 5)
gen.add("Workspace_X_Position", double_t, 0, "Workspace_X_Position", 0, -5, 5)
gen.add("Workspace_Y_Position", double_t, 0, "Workspace_Y_Position", 0, -5, 5)

exit(gen.generate(PACKAGE, "BaseServoPID", "BaseServoPID"))
